source/_components/unifi.markdown

```diff
@@ -37,9 +37,10 @@ unifi:
   controllers:
     - host: unifi
       site: My site
+      dont_track_devices: true
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Fixed!